consensus/polybft/consensus_runtime.go

```diff
@@ -648,7 +648,7 @@ func (c *consensusRuntime) getAggSignatureForCommitmentMessage(
 
 	publicKeys := make([][]byte, 0)
 	bitmap := bitmap.Bitmap{}
-	signers := make([]types.Address, 0)
+	signers := make(map[types.Address]struct{}, 0)
```

Comment:
you can use votes length there

Reply:
I think he cannot, because not all the votes are always included, since if for **some** reason sender doesn't exist in `validatorAddrToIndex`, that signer just doesn't get included to the `signers` map.